python39Packages.pywebview tests failing

### Describe the bug
`python39Packages.pywebview` is failing to build due to test failures

### Steps To Reproduce
Steps to reproduce the behavior:
1. `nix build github:NixOS/nixpkgs/master#python39Packages.pywebview`
```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing setuptools-check-hook
@nix { "action": "setPhase", "phase": "qtPreHook" }
qtPreHook
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/2wa4gn21wyvfqnp6bkv875ad6y9r0swl-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/windows_interop/WebBrowserInterop.sln
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
Executing setuptoolsBuildPhase
running bdist_wheel
running build
running build_py
creating build
creating build/lib
creating build/lib/webview
copying webview/localization.py -> build/lib/webview
copying webview/screen.py -> build/lib/webview
copying webview/serving.py -> build/lib/webview
copying webview/wsgi.py -> build/lib/webview
copying webview/__init__.py -> build/lib/webview
copying webview/event.py -> build/lib/webview
copying webview/guilib.py -> build/lib/webview
copying webview/util.py -> build/lib/webview
copying webview/window.py -> build/lib/webview
creating build/lib/webview/js
copying webview/js/__init__.py -> build/lib/webview/js
copying webview/js/alert.py -> build/lib/webview/js
copying webview/js/css.py -> build/lib/webview/js
copying webview/js/dom.py -> build/lib/webview/js
copying webview/js/drag.py -> build/lib/webview/js
copying webview/js/event.py -> build/lib/webview/js
copying webview/js/npo.py -> build/lib/webview/js
copying webview/js/api.py -> build/lib/webview/js
creating build/lib/webview/platforms
copying webview/platforms/__init__.py -> build/lib/webview/platforms
copying webview/platforms/cef.py -> build/lib/webview/platforms
copying webview/platforms/cocoa.py -> build/lib/webview/platforms
copying webview/platforms/edgechromium.py -> build/lib/webview/platforms
copying webview/platforms/edgehtml.py -> build/lib/webview/platforms
copying webview/platforms/gtk.py -> build/lib/webview/platforms
copying webview/platforms/mshtml.py -> build/lib/webview/platforms
copying webview/platforms/qt.py -> build/lib/webview/platforms
copying webview/platforms/winforms.py -> build/lib/webview/platforms
running egg_info
creating pywebview.egg-info
writing pywebview.egg-info/PKG-INFO
writing dependency_links to pywebview.egg-info/dependency_links.txt
writing requirements to pywebview.egg-info/requires.txt
writing top-level names to pywebview.egg-info/top_level.txt
writing manifest file 'pywebview.egg-info/SOURCES.txt'
reading manifest file 'pywebview.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE.md'
writing manifest file 'pywebview.egg-info/SOURCES.txt'
creating build/lib/webview/lib
copying webview/lib/Microsoft.Toolkit.Forms.UI.Controls.WebView.LICENSE.md -> build/lib/webview/lib
copying webview/lib/Microsoft.Toolkit.Forms.UI.Controls.WebView.dll -> build/lib/webview/lib
copying webview/lib/Microsoft.Web.WebView2.Core.dll -> build/lib/webview/lib
copying webview/lib/Microsoft.Web.WebView2.LICENSE.md -> build/lib/webview/lib
copying webview/lib/Microsoft.Web.WebView2.WinForms.dll -> build/lib/webview/lib
copying webview/lib/WebBrowserInterop.x64.dll -> build/lib/webview/lib
copying webview/lib/WebBrowserInterop.x86.dll -> build/lib/webview/lib
creating build/lib/webview/lib/x64
copying webview/lib/x64/WebView2Loader.dll -> build/lib/webview/lib/x64
creating build/lib/webview/lib/x86
copying webview/lib/x86/WebView2Loader.dll -> build/lib/webview/lib/x86
/nix/store/xi8ha1k3mjpmnmmi5fp7fmq41f2wwghm-python3.9-setuptools-61.2.0/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/localization.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/screen.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/serving.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/wsgi.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/__init__.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/event.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/guilib.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/util.py -> build/bdist.linux-x86_64/wheel/webview
copying build/lib/webview/window.py -> build/bdist.linux-x86_64/wheel/webview
creating build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/__init__.py -> build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/alert.py -> build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/css.py -> build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/dom.py -> build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/drag.py -> build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/event.py -> build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/npo.py -> build/bdist.linux-x86_64/wheel/webview/js
copying build/lib/webview/js/api.py -> build/bdist.linux-x86_64/wheel/webview/js
creating build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/__init__.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/cef.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/cocoa.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/edgechromium.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/edgehtml.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/gtk.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/mshtml.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/qt.py -> build/bdist.linux-x86_64/wheel/webview/platforms
copying build/lib/webview/platforms/winforms.py -> build/bdist.linux-x86_64/wheel/webview/platforms
creating build/bdist.linux-x86_64/wheel/webview/lib
copying build/lib/webview/lib/Microsoft.Toolkit.Forms.UI.Controls.WebView.LICENSE.md -> build/bdist.linux-x86_64/wheel/webview/lib
copying build/lib/webview/lib/Microsoft.Toolkit.Forms.UI.Controls.WebView.dll -> build/bdist.linux-x86_64/wheel/webview/lib
copying build/lib/webview/lib/Microsoft.Web.WebView2.Core.dll -> build/bdist.linux-x86_64/wheel/webview/lib
copying build/lib/webview/lib/Microsoft.Web.WebView2.LICENSE.md -> build/bdist.linux-x86_64/wheel/webview/lib
copying build/lib/webview/lib/Microsoft.Web.WebView2.WinForms.dll -> build/bdist.linux-x86_64/wheel/webview/lib
copying build/lib/webview/lib/WebBrowserInterop.x64.dll -> build/bdist.linux-x86_64/wheel/webview/lib
copying build/lib/webview/lib/WebBrowserInterop.x86.dll -> build/bdist.linux-x86_64/wheel/webview/lib
creating build/bdist.linux-x86_64/wheel/webview/lib/x64
copying build/lib/webview/lib/x64/WebView2Loader.dll -> build/bdist.linux-x86_64/wheel/webview/lib/x64
creating build/bdist.linux-x86_64/wheel/webview/lib/x86
copying build/lib/webview/lib/x86/WebView2Loader.dll -> build/bdist.linux-x86_64/wheel/webview/lib/x86
running install_egg_info
Copying pywebview.egg-info to build/bdist.linux-x86_64/wheel/pywebview-3.6.1-py3.9.egg-info
running install_scripts
adding license file "LICENSE.md" (matched pattern "LICEN[CS]E*")
creating build/bdist.linux-x86_64/wheel/pywebview-3.6.1.dist-info/WHEEL
creating 'dist/pywebview-3.6.1-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'webview/__init__.py'
adding 'webview/event.py'
adding 'webview/guilib.py'
adding 'webview/localization.py'
adding 'webview/screen.py'
adding 'webview/serving.py'
adding 'webview/util.py'
adding 'webview/window.py'
adding 'webview/wsgi.py'
adding 'webview/js/__init__.py'
adding 'webview/js/alert.py'
adding 'webview/js/api.py'
adding 'webview/js/css.py'
adding 'webview/js/dom.py'
adding 'webview/js/drag.py'
adding 'webview/js/event.py'
adding 'webview/js/npo.py'
adding 'webview/lib/Microsoft.Toolkit.Forms.UI.Controls.WebView.LICENSE.md'
adding 'webview/lib/Microsoft.Toolkit.Forms.UI.Controls.WebView.dll'
adding 'webview/lib/Microsoft.Web.WebView2.Core.dll'
adding 'webview/lib/Microsoft.Web.WebView2.LICENSE.md'
adding 'webview/lib/Microsoft.Web.WebView2.WinForms.dll'
adding 'webview/lib/WebBrowserInterop.x64.dll'
adding 'webview/lib/WebBrowserInterop.x86.dll'
adding 'webview/lib/x64/WebView2Loader.dll'
adding 'webview/lib/x86/WebView2Loader.dll'
adding 'webview/platforms/__init__.py'
adding 'webview/platforms/cef.py'
adding 'webview/platforms/cocoa.py'
adding 'webview/platforms/edgechromium.py'
adding 'webview/platforms/edgehtml.py'
adding 'webview/platforms/gtk.py'
adding 'webview/platforms/mshtml.py'
adding 'webview/platforms/qt.py'
adding 'webview/platforms/winforms.py'
adding 'pywebview-3.6.1.dist-info/LICENSE.md'
adding 'pywebview-3.6.1.dist-info/METADATA'
adding 'pywebview-3.6.1.dist-info/WHEEL'
adding 'pywebview-3.6.1.dist-info/top_level.txt'
adding 'pywebview-3.6.1.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Finished executing setuptoolsBuildPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
glibPreInstallPhase
@nix { "action": "setPhase", "phase": "installPhase" }
installing
Executing pipInstallPhase
/build/source/dist /build/source
Processing ./pywebview-3.6.1-py3-none-any.whl
Requirement already satisfied: proxy-tools in /nix/store/gsyf8h877j5gxal9xy5n3fs8gachq6h3-python3.9-proxy_tools-0.1.0/lib/python3.9/site-packages (from pywebview==3.6.1) (0.1.0)
Installing collected packages: pywebview
Successfully installed pywebview-3.6.1
/build/source
Finished executing pipInstallPhase
@nix { "action": "setPhase", "phase": "qtOwnPathsHook" }
qtOwnPathsHook
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
glibPreFixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
post-installation fixup
wrapping Qt applications in /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1/bin /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1/sbin /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1/libexec /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1/Applications
shrinking RPATHs of ELF executables and libraries in /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1
strip is /nix/store/58pwclg9yr437h0pfgrnbd0jis8fqasd-gcc-wrapper-11.2.0/bin/strip
stripping (with command strip and flags -S) in /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1/lib 
patching script interpreter paths in /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1
checking for references to /build/ in /nix/store/481la7zxg73bq7b9vhs80khg1k0n9qbl-python3.9-pywebview-3.6.1...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
@nix { "action": "setPhase", "phase": "installCheckPhase" }
running install tests
~/source/tests ~/source
patching script interpreter paths in run.sh
run.sh: interpreter directive changed from "#!/bin/bash" to "/nix/store/fcd0m68c331j7nkdxvnnpb8ggwsaiqac-bash-5.1-p16/bin/bash"
rm: cannot remove '__pycache__': No such file or directory
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
ImportError while loading conftest '/build/source/tests/conftest.py'.
conftest.py:2: in <module>
    from six.moves import reload_module
E   ModuleNotFoundError: No module named 'six'
TEST FAILURES HAVE OCCURRED!

```

### Expected behavior
Package should buikd

### Notify maintainers
@jojosch

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.17.1, NixOS, 22.05 (Quokka), 22.05.20220330.710fed5`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.0pre20220401_bf48959`
 - channels(root): `""`
```

--- a/pkgs/development/python-modules/pywebview/default.nix
+++ b/pkgs/development/python-modules/pywebview/default.nix
@@ -2,24 +2,29 @@
 , buildPythonPackage
 , fetchFromGitHub
 , importlib-resources
+, proxy_tools
+, pygobject3
 , pyqtwebengine
 , pytest
 , pythonOlder
 , qt5
+, qtpy
+, six
 , xvfb-run
-, proxy_tools
 }:
 
 buildPythonPackage rec {
   pname = "pywebview";
-  version = "3.6.1";
+  version = "3.6.3";
+  format = "setuptools";
+
   disabled = pythonOlder "3.5";
 
   src = fetchFromGitHub {
     owner = "r0x0r";
     repo = "pywebview";
     rev = version;
-    sha256 = "sha256-9o9ghqvU9Hnmf2aj/BqX7WBgS9ilRSnicR+qd25OfjI=";
+    hash = "sha256-qOLK4MHdpmcCazCNfojncD8XH7OJB2H/pIW5XAJAlDo=";
   };
 
   nativeBuildInputs = [
@@ -29,10 +34,15 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     pyqtwebengine
     proxy_tools
-  ] ++ lib.optionals (pythonOlder "3.7") [ importlib-resources ];
+    six
+  ] ++ lib.optionals (pythonOlder "3.7") [
+    importlib-resources
+  ];
 
   checkInputs = [
+    pygobject3
     pytest
+    qtpy
     xvfb-run
   ];
 
@@ -54,11 +64,13 @@ buildPythonPackage rec {
     popd
   '';
 
-  pythonImportsCheck = [ "webview" ];
+  pythonImportsCheck = [
+    "webview"
+  ];
 
   meta = with lib; {
-    homepage = "https://github.com/r0x0r/pywebview";
     description = "Lightweight cross-platform wrapper around a webview";
+    homepage = "https://github.com/r0x0r/pywebview";
     license = licenses.bsd3;
     maintainers = with maintainers; [ jojosch ];
   };

